README.md

```diff
@@ -12,12 +12,12 @@ This is the repository for the [Decidim Documentation website](https://docs.deci
 [Install antora](https://docs.antora.org/antora/2.3/install-and-run-quickstart/) with software dependencies.
 
 ```bash
-# Install Node & npm using nvm
+# Install Node an Npm, for instance using nvm:
 wget -qO- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash
 nvm install node
 
-# Install Antora and dependencies
-npm install
+# Install Antora and dependencies (some system will need to prefix `sudo`)
+npm install -g antora
```

Comment:
There are some other [packages](https://github.com/decidim/documentation/blob/master/package.json) for this to work as the production instance. Does the installation not work with `npm install`? Do you have a traceback or something?

Reply:
As a matter of fact, maybe is not necessary to have installed antora globally. In any case npm install will only work if it is executed inside the documention folder, which at this step is not cloned yet.

Reply:
Wow, I hadn't noticed that. You're absolutely right, impossible that this works as it is now.

Reply:
Can you move this to after line 26 ( `cd documentation/`) and leave it as `npm install` 🙏🏽?

Reply:
done!